[BUG]:   Build error with DTYPE_OBJECT flag for esp32

Build process does not complete for esp32 if flag for DTYPE_OBJECT is set in ulab.h:
#define ULAB_HAS_DTYPE_OBJECT               (1)

Error following execution of make command:
/($BUILD_DIR)/ulab/code/ulab.c:182:5: error: expected declaration specifiers or '...' before 'ulab_dtype_type'
     ulab_dtype_type,
     ^~~~~~~~~~~~~~~
No other flags from the default ulab.h file were set for this build.

--- a/code/ulab.c
+++ b/code/ulab.c
@@ -182,7 +182,7 @@ MP_DEFINE_CONST_OBJ_TYPE(
     ulab_dtype_type,
     MP_QSTR_dtype,
     MP_TYPE_FLAG_NONE,
-    print, ndarray_dtype_print
+    print, ndarray_dtype_print,
     make_new, ndarray_dtype_make_new
 );
 #else

